hellochain/tutorial/15-full-app.md

```diff
@@ -11,4 +11,4 @@ and add greeter's AppModuleBasic and AppModule alongside the other modules.
 
 Update your `app.go` to look like the following
 
-<<< @/hellochain/app.go{12,25,26,33,36,39,44,45,49,50,53}
```

Comment:
is there anyway we can keep the line highlights? they are pretty useful

Reply:
We're using `{...}` syntax for annotating things like synopsis and prerequisite links, so we can't use it for line numbers. I'm thinking of using `.../app.go[12,25,26]` syntax instead.